bundles/org.openhab.binding.amplipi/README.md

```diff
@@ -0,0 +1,91 @@
+# AmpliPi Binding
+
+This binding supports the multi room audio system [AmpliPi](http://www.amplipi.com/) from [MicroNova](http://www.micro-nova.com/).
+
+
+## Supported Things
+
+The AmpliPi itself is modeled as a Bridge of type `controller`.
+Every available zone as well as group is managed as an individual Thing of type `zone` resp. `group`.
+
+## Discovery
+
+Once the AmpliPi announces itself through mDNS (still a pending feature), it will be automatically discovered on the network.
+
+As soon as the AmpliPi is online, its zones and groups are automatically retrieved and added as Things to the Inbox.
+
+## Thing Configuration
+
+The `controller` Bridge has two configuration parameters:
+
+| Parameter       | Required | Description                                                                                        |
+|-----------------|----------|----------------------------------------------------------------------------------------------------|
+| hostname        | yes      | The hostname or IP address of the AmpliPi on the network                                           |
+| refreshInterval | no       | The time to wait between two polling requests for receiving state updates. Defaults to 10 seconds. |
+
+Both the `zone` and `group` Things only require a single configuration parameter `id`, which corresponds to their id on the AmpliPi.
+
+## Channels
+
+These are the channels of the `controller` Bridge:
+
+| Channel  | Type   | Description                                                                                          |
+|----------|--------|------------------------------------------------------------------------------------------------------|
+| preset   | Number | Allows setting a pre-configured preset. The available options are dynamically read from the AmpliPi. |
+| input1   | String | The selected input of source 1                                                                       |
```

Comment:
Is the type correct for `input1-4`? Should be `Number` based on your example below?